controllers/products.ts

```diff
@@ -15,6 +15,19 @@ interface GetParams {
   query?: string
 }
 
+export const getProduct = async (
+  productId: string | string[],
```

Comment:
This seems incorrect

Reply:
the default typing is as follows

![imagen](https://user-images.githubusercontent.com/54547525/191273454-5e8ab312-d2d6-43ac-afc9-a3d59e9ee751.png)

Reply:
the id is also sent in this way in product elimination, with this typeface

![imagen](https://user-images.githubusercontent.com/54547525/191274552-b50deb20-e3bf-4e07-90f2-cd425d43df1c.png)

Reply:
Hmm to me it should be only string

@e-azocar please check